docs/03-tutorials/00-serverless/svls-15-use-external-scalers.md

```diff
@@ -0,0 +1,137 @@
+
+- Kyma installed on a cluster
+
+## Steps
+
+Follow these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="hpa">
+  HorizontalPodAutoscaler
+  </summary>
+
+1. Create the Function with the `replicas` value set to 1 to prevent the internal Serverless HorizontalPodAutoscaler creation:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha2
+    kind: Function
+    metadata:
+      name: scaled-function
+    spec:
+      runtime: nodejs14
+      replicas: 1
+      source:
+        inline:
+          dependencies: ""
+          source: |
+            module.exports = {
+              main: function(event, context) {
+                return 'Hello World!'
+              }
+            }
+    EOF
+    ```
+
+2. Create the HorizontalPodAutoscaler using the kubectl:
+
+    ```bash
+    kubectl autoscale function scaled-function --cpu-percent=50 --min=5 --max=10
+    ```
+
+3. After a few seconds the HorizontalPodAutoscaler should be up to date and contain information about actual replicas:
+
+    ```bash
+    kubectl get hpa scaled-function
+    ```
+
+    You should get a result similar to this example:
+
+    ```bash
+    NAME              REFERENCE                  TARGETS   MINPODS   MAXPODS   REPLICAS   AGE
+    scaled-function   Function/scaled-function   1%/50%    5         10        5          61s
+    ```
+
+  </details>
+  <details>
+  <summary label="keda">
+  Keda
+  </summary>
+
+1. Install [Keda](https://keda.sh/docs/2.8/deploy/) if it does not present on your cluster.
+
+2. Create the Function with the `replicas` value set to 1 to prevent the internal Serverless HorizontalPodAutoscaler creation:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha2
+    kind: Function
+    metadata:
+      name: scaled-function
+    spec:
+      runtime: nodejs14
+      replicas: 1
+      source:
+        inline:
+          dependencies: ""
+          source: |
+            module.exports = {
+              main: function(event, context) {
+                return 'Hello World!'
+              }
+            }
+    EOF
+    ```
+
+3. Create the ScaledObject resource:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: keda.sh/v1alpha1
+    kind: ScaledObject
+    metadata:
+      name: scaled-function
+    spec:
+      scaleTargetRef:
+        apiVersion:    serverless.kyma-project.io/v1alpha2
+        kind:          Function
+        name:          scaled-function
+      minReplicaCount:  5
+      maxReplicaCount:  10
+      triggers:
+      - type: cpu
+        metricType: Utilization
+        metadata:
+          value: "50"
+    EOF
+    ```
+
+    >**NOTE:** in this tutorial we use the `cpu` trigger because its configuration is really easy. If you want to use another trigger check the official [list of supported triggers](https://keda.sh/docs/2.8/scalers/).
```

Comment:
```suggestion
    >**NOTE:** in this tutorial we use the `cpu` trigger because of its simple configuration. If you want to use another trigger check the official [list of supported triggers](https://keda.sh/docs/2.8/scalers/).
```